iam/api-client/src/test/java/iam/snippets/ServiceAccountTests.java

```diff
@@ -71,7 +72,7 @@ public void tearDown() {
   }
 
   @Test
-  public void stage1_testServiceAccountCreate() {
+  public void stage1_testServiceAccountCreate() throws IOException {
     CreateServiceAccount.createServiceAccount(PROJECT_ID, SERVICE_ACCOUNT);
     String got = bout.toString();
     assertThat(got, containsString("Created service account: " + SERVICE_ACCOUNT));
```

Comment:
Assert using the returned value from ` CreateServiceAccount.createServiceAccount`